Update interfaces/html.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/32e76fe/whatwg/idl/html.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/480736233

foolip commented:
This will have to be done manually, taking changes from https://github.com/web-platform-tests/wpt/pull/11631 and running them through Chromium's CQ first to make sure there aren't any regressions.

foolip commented:
Also blocked by https://github.com/tidoust/reffy/issues/129.

foolip commented:
I'm doing a manual update in https://github.com/web-platform-tests/wpt/pull/14886 so that we're not so far behind the spec.

autofoolip committed:
Update interfaces/html.idl

Source: https://github.com/tidoust/reffy-reports/blob/32e76fe/whatwg/idl/html.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/480736233
